data/net/minecraft/world/entity/ai/memory/MemoryModuleType.mapping

```diff
@@ -1,4 +1,12 @@
 CLASS net/minecraft/world/entity/ai/memory/MemoryModuleType
+	FIELD NEAREST_LIVING_ENTITIES Lnet/minecraft/world/entity/ai/memory/MemoryModuleType;
+		COMMENT Holds a list of nearby living entities inside the
+		COMMENT brain entity's bounding box expanded 16 blocks
+		COMMENT in all directions.
+	FIELD NEAREST_VISIBLE_LIVING_ENTITIES Lnet/minecraft/world/entity/ai/memory/MemoryModuleType;
+		COMMENT Holds a list of nearby living entities inside the
+		COMMENT brain entity's bounding box expanded 16 blocks
+		COMMENT in all directions which the brain entity can see.
```

Comment:
These comments should all be on one line as they are one sentence. Refer to Javadoc Standards 2.